### DISCUSSION TITLE
What is and how to use "projects:[]" in boss.json?

### DISCUSSION THREAD

**Discussion by sbeu** [HashLoad/boss#141] [Q&A]
Hello,
I see in the `boss.json` file this entry:
projects:[]

What is the use of this entry?

My guess: We can have one git repository and inside this git repository, have various "packages". If it is the case, how to use it?

Thank you

**Comment by sbeu** [comment]
Is there no use for this entry??

**Comment by andriwsluna** [comment]
As a dependency, this is used to define which packages will be built, so that .dcus can be generated. As a dependent, this is used to know in which projects the appropriate searchpaths will be added.

**Reply by perettigiuliano** [reply]
Hi, sorry but it is not clear, at least for me.
Could you point out some sort of documentation?
Thanks
